src/site/_data/i18n/common.yml

```diff
@@ -1,55 +1,143 @@
 note:
   en: Note
+  es: Nota
+  ja: メモ
+  ko: 메모
+  pt: Observação
+  ru: Примечание
+  zh: 注意
 
 caution:
   en: Caution
+  es: Precaución
+  ja: 注意
+  ko: 주의
+  pt: Cuidado
+  ru: Внимание
+  zh: 小心
 
 warning:
   en: Warning
+  es: Advertencia
+  ja: 警告
+  ko: 경고
+  pt: Aviso
+  ru: Предупреждение
+  zh: 警告
 
 success:
   en: Success
+  es: Éxito
+  ja: 成功
+  ko: 성공
+  pt: Sucesso
+  ru: Успех
+  zh: 成功
 
 objective:
   en: Objective
+  es: Objetivo
+  ja: 目的
+  ko: 목적
+  pt: Objetivo
+  ru: Цель
+  zh: 目标
 
 try_it:
   en: Try it
+  es: Pruebelo
+  ja: お試しください
+  ko: 시도하기
+  pt: Experimente
+  ru: Попробуйте
+  zh: 请尝试
 
 key_term:
   en: Key Term
+  es: Término clave
+  ja: 重要な用語
+  ko: 핵심 용어
+  pt: Termo chave
+  ru: Ключевой термин
+  zh: 重要词汇
 
 gotchas:
   en: Gotchas
   es: Gotchas
+  ja: 問題の原因
+  ko: 갓차
+  pt: Pegadinhas
+  ru: Обратите внимание
+  zh: 问题
 
 do:
   en: Do
   es: Hacer
```

Comment:
Do and Don't are like a pair, so should be in the same verb form. I think Hacer is an infinitive instead of 2st person form (No hagas).